src/encoded/static/components/filegallery.js

```diff
@@ -484,7 +484,7 @@ export class FileTable extends React.Component {
 
             return (
                 <div>
-                    {showFileCount ? <div className="file-gallery-counts">Displaying {filteredCount} of {unfilteredCount} files</div> : null}
+                    {showFileCount ? <div className="table-counts">Displaying {filteredCount} of {unfilteredCount} files</div> : null}
```

Comment:
Is `table-counts` descriptive enough? `file-gallery-counts` tells me more, IMHO.

Reply:
It applied to tables that aren’t the file gallery.